api/gateway.proto

```diff
@@ -432,3 +432,8 @@ message GatewayConnectionStats {
 
   // next: 13
 }
+
+message GatewayConnectionStatsEntry{
+  GatewayIdentifiers gateway_ids = 1;
+  GatewayConnectionStats gateway_connection_stats = 2;
+}
```

Comment:
You can also move this message inside `BatchGetGatewayConnectionStatsResponse`

Reply:
That would make the type a bit long to use; `ttnpb.BatchGetGatewayConnectionStatsResponse_GatewayConnectionStatsEntry`.